bundle-community-operators/manifests/datadog-operator-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I wonder why this file is here, since it's not in the bundle/manifests directory..

Reply:
A whole `bundle-community-operators` mirroring `bundle` directory got added wen I ran `make bundle`. I'll look into it most likely with @fanny-jiang

Reply:
It's expected, is a requirement for community operator bundle. It will be committed on the release branch for the job to pick it up, but not in `main`